integration/setup/k8s/setup.go

```diff
@@ -13,6 +13,14 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+/*
+	NOTE: This code is copied from e2esetup/k8s package to avoid circular
+	dependencies. All changes made there should be reflected in e2esetup.
+
+		See https://github.com/giantswarm/e2esetup/pull/13
+
+*/
```

Comment:
The diff is small because git thinks the file was moved from vendor directory. This is a full copy.